sdk/textanalytics/Azure.AI.TextAnalytics/tests/AnalyzeOperationTests.cs

```diff
@@ -172,31 +174,44 @@ public async Task AnalyzeOperationWithMultipleTasks()
                 }
             };
 
-            AnalyzeOperationOptions operationOptions = new AnalyzeOperationOptions()
+            TextAnalyticsActions batchActions = new TextAnalyticsActions()
             {
-                KeyPhrasesTaskParameters = new KeyPhrasesTaskParameters(),
-                EntitiesTaskParameters = new EntitiesTaskParameters(),
-                PiiTaskParameters = new PiiTaskParameters(),
+                ExtractKeyPhrasesOptions = new List<ExtractKeyPhrasesOptions>() { new ExtractKeyPhrasesOptions() },
+                RecognizeEntitiesOptions = new List<RecognizeEntitiesOptions>() { new RecognizeEntitiesOptions() },
+                RecognizePiiEntitiesOptions = new List<RecognizePiiEntitiesOptions>() { new RecognizePiiEntitiesOptions() },
                 DisplayName = "AnalyzeOperationWithMultipleTasks"
             };
 
-            AnalyzeOperation operation = await client.StartAnalyzeOperationBatchAsync(batchDocuments, operationOptions);
+            AnalyzeBatchActionsOperation operation = await client.StartAnalyzeBatchActionsAsync(batchDocuments, batchActions);
+
+            Assert.AreEqual(operation.ActionsFailed, 0);
+            Assert.AreEqual(operation.ActionsSucceeded, 0);
+            Assert.AreEqual(operation.ActionsInProgress, 0);
+            Assert.AreEqual(operation.TotalActions, 0);
 
             await operation.WaitForCompletionAsync(PollingInterval);
 
+            Assert.AreEqual(operation.ActionsFailed, 0);
+            Assert.AreEqual(operation.ActionsSucceeded, 3);
+            Assert.AreEqual(operation.ActionsInProgress, 0);
+            Assert.AreEqual(operation.TotalActions, 3);
+            Assert.IsNotNull(operation.ExpiresOn);
+            Assert.IsNotNull(operation.CreatedOn);
```

Comment:
it will be more accurate to check that they are not default. see https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/textanalytics/Azure.AI.TextAnalytics/tests/RecognizeHealthcareEntitiesTests.cs#L392